python/seldon_core/batch_processor.py

```diff
@@ -258,6 +271,13 @@ def _start_request_worker(
                 batch_id,
             )
             q_out.put(str_output)
+
+        # Setting time interval before the task is marked as done
+        if batch_interval > 0:
+            remaining_interval = batch_interval - (time.time() - start_time)
+            if remaining_interval > 0:
+                time.sleep(remaining_interval)
```

Comment:
@RafalSkolasinski I have updated the interval as the minimum time interval between batch prediction tasks